k8s-install/canal-kubernetes-datastore.yaml

```diff
@@ -23,6 +23,7 @@ data:
         "type": "calico",
```

Comment:
Is it enough of a risk to maintain the `canal-etcdless` name?  We're trying to move to remove the `etcdless` term from the KDD stuff.